packages/core-components/src/layout/Sidebar/Items.tsx

```diff
@@ -290,3 +290,11 @@ export const SidebarDivider = styled('hr')({
   border: 'none',
   margin: '12px 0px',
 });
+
+export const SidebarScrollWrapper = styled('div')({
+  flex: '0 1 auto',
+  overflowY: 'auto',
+  // Display at least one item in the container
+  // Question: Can this be a config/theme variable - if so, which? :/
+  minHeight: '48px',
```

Comment:
Hmm not at this point it seems, SidebarItem has a [fixed size of 48](https://github.com/backstage/backstage/blob/fe2b2d4b5fd950656b4f5f8e3e83228a89a1640e/packages/core-components/src/layout/Sidebar/Items.tsx#L52) so you are mirroring that fine right now.